src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,43 @@
-export const Tabs = () => {};
+import React from 'react';
+import classNames from 'classnames';
+
+export const Tabs = ({
+  tabs,
+  selectedTabId,
+  onTabSelected,
+}) => {
+  const activeTab = tabs.find(tab => tab.id === selectedTabId) || tabs[0];
+
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map((tab) => {
+            const isCurrentTabActive = activeTab.id === tab.id;
+
+            return (
+              <li
+                key={tab.id}
+                data-cy="Tab"
+                className={classNames({ 'is-active': isCurrentTabActive })}
+              >
+
+                <a
+                  href={`#${tab.id}`}
+                  data-cy="TabLink"
+                  onClick={selectTab}
```

Comment:
Fix this error. Maybe you need `onTabSelected`?
![image](https://github.com/mate-academy/react_tabs-js/assets/70009205/d16345cf-de7d-48c8-b4e8-5d7a55e06703)